app/controllers/incident_reports_controller.rb

```diff
@@ -32,6 +33,17 @@ def build_passengers
     @report.injured_passengers.build unless @report.injured_passengers.present?
     @report.injured_passengers
   end
+  
+  def delete_passengers
+    if report_params[:injured_passengers_attributes]
+      report_params[:injured_passengers_attributes].each do |pax_num, pax_info|
+        # only their id is given, which means that they were deleted.
+        if pax_info.values.length == 1
+          @report.injured_passengers.destroy(pax_info.values.first)
```

Comment:
This required some pretty close reading in order to parse... but I see the necessity and I'm not sure how to do it differently.